coordinator/src/services/communication/tracker/sandbox.ts

```diff
@@ -35,7 +35,7 @@ export class SandboxTrackerClient implements TrackerClient {
             totalAuthorised: "1"
           },
           pendingCancellations: "False",
-          prescriptionTreatmentType: "Acute Prescribing",
+          prescriptionTreatmentType: "Acute Prescription",
```

Comment:
Confirmed this is correct :+1: must have got the original example from the spec, how silly of me

Reply:
lol